helm/publisher/templates/article-page-deployment.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}--article-page
+  labels:
+    {{- include "publisher.labels" . | nindent 4 }}
+    app.kubernetes.io/component: article-page
+spec:
+  selector:
+    matchLabels:
+      {{- include "publisher.selectorLabels" . | nindent 6 }}
+      app.kubernetes.io/component: article-page
+  template:
+    metadata:
+      labels:
+        {{- include "publisher.selectorLabels" . | nindent 8 }}
+        app.kubernetes.io/component: article-page
+    spec:
+      containers:
+        - name: app
+          image: "{{ .Values.images.articlePage.repository }}:latest"
```

Comment:
Speccing 'latest' in the template feels wrong to me. Shouldn't this live in `values.yaml` and ideally not use latest?

Reply:
Yes, this should be a fixed version, but it is to keep consistency with https://github.com/libero/publisher-chart/blob/master/helm/publisher/templates/article-store-deployment.yaml#L21. Separately I expect to pin the tags of both and to find a way to update them in a controlled way (a la Dependabot/Renovate).